pkg/loop/internal/pb/mercury/v4/datasource_v4.proto

```diff
@@ -28,8 +28,6 @@ message Block {
 // Observation is a gRPC adapter for [pkg/types/mercury/v1/Observation]
 message Observation {
     BigInt benchmarkPrice = 1;
-    BigInt bid = 2;
-    BigInt ask = 3;
```

Comment:
Should we mark them deprecated? IIRC there are two different ways to do it

Reply:
We can mark them as deprecated, but since they've never been used in a deployed environment, I thought it would be easier to simply remove them for now.

Reply:
Removing is fine, but we should reserve them instead of leaving a gap without explanation:
https://protobuf.dev/programming-guides/proto3/#deleting

Otherwise deprecation is an 'option':
https://protobuf.dev/programming-guides/proto3/#options

Reply:
I added the fields back with a deprecated tag.